src/styles/styles.js

```diff
@@ -777,6 +777,14 @@ const styles = {
         borderColor: themeColors.danger,
     },
 
+    textInputDisabled: {
+        backgroundColor: themeColors.overlay,
+        borderColor: themeColors.borderLighter,
+        WebkitTextFillColor: themeColors.textSupporting,
+        WebkitOpacity: 1,
```

Comment:
Can you add a comment explaining why we need this?

Also elaborate how this solves our issue thanks!